_posts/2016/2016-04-21-jenadevs-docker-party.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "First Jena Devlopers Meetup: Docker Party!"
+date: "2016-04-11 13:37:42"
+icon: cubes
+categories: events
+authors: ["Benjamin and Oliver Z."]
+---
+
+On the first Thursday of April the newly founded [Softwerkskammer Jena](https://www.softwerkskammer.org/groups/jena) had its first [Developers Meetup](www.meetup.com/jenadevs).
+ePages took the opportunity to be the event host and provide not just conference rooms and serveral workstation laptops but also free pizza, snacks, [Club-Mate](https://de.wikipedia.org/wiki/Club-Mate) and some other refreshments for everyone.
+Despite the short notice over 30 attendees from a dozen of companies from the area of Jena, university students and even some freelancers followed the invitation to the Docker Party.
+They joined the orga team of 8 developers for 4 hours of coding fun from 6 pm until nearly 10 pm with an open end at the local pub, where the devoted discussions lasted until 2 am in the morning.
+
+## Agenda
+
+At the beginning there was a short period of spare time for meet and greet as well as a brief introduction about the [Softwerkskammer](softwerkskammer.org) – the german part of the [Software Craftsmanship Community](http://manifesto.softwarecraftsmanship.org) – and its current distribution with some helpful contact information like the [@jenadevs](https://twitter.com/jenadevs) account on Twitter.
+
+Then the Docker Party, which celebrated the [3rd birthday of Docker](https://www.docker.com/community/docker-birthday-3), was launched! 
+It consisted of an one hour talk including a demo to introduce the technological foundation and nearly three hours of different workshops for every skill level:
+
+  1. **Beginner:** The official Docker birthday app tutorial
+  1. **Advanced:** Building your own Dockerfile accoriding to best practises in conjunction with integration tests 
+  1. **Special:** CoresOS Cluster Workshop
+  1. **Hidden - also known as exploding kittens:** Orchestration Workshop by [Jérôme Petazzoni](https://twitter.com/jpetazzo), the creator of [Docker-in-Docker](https://github.com/jpetazzo/dind) 
+
+## Introduction talk with demo and best practises
+
+In the first place the talk focused on the Docker basics, the components of the Docker ecosystem and the general tooling workflow of building images, running containers and pushing them to the Docker Hub.
+Afterwards the basic commands were demonstrated with some live coding on the terminal, where several containers were also inspected at runtime.
```

Comment:
We had a demo on the basic commands along with some live coding on the terminal, where...